compiler-core/src/erlang/tests/records.rs

```diff
@@ -49,6 +49,7 @@ fn module_types() {
             "name",
             Arc::new(type_::Type::Named {
                 publicity: Publicity::Public,
+                from_internal_module: false,
```

Comment:
Isn't this duplicating the publicity field? It has a `Publicity::Internal` variant

Reply:
I thought about it but then the formatter would then put the `@internal` annotation above everything inside an internal module

Reply:
The formatter doesn't run analysis so it wouldn't know the module is internal

Reply:
Ah I didn't think about it! Thanks I've removed the redundant field